config/routes.rb

```diff
@@ -421,4 +421,5 @@ def paginate_url(url, params)
   post '/api/v1/send_sms', to: 'api/v1/send_sms#index'
   post '/api/v1/sms_configuration', to: 'api/v1/send_sms#update'
   get '/api/v1/configurations', to: 'api/v1/send_sms#show'
+  get '/api/v1/patient_phone', to: 'api/v1/send_sms#fetch_phone'
```

Comment:
Just thinking isn't the clients phone number already available in the frontend by the time we would like to send an sms or the time we are trying to show the send sms popup